javascript/ember/app/.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
```

Comment:
Do we need this file?

Reply:
Ah, no we don't. The `ember new` command adds a lot of stuff we don't really need. Thank you for noticing!